ntp/init.sls

```diff
@@ -11,3 +12,14 @@ ntp:
     - reload: True
     - watch:
       - pkg: ntp
+{% else %}
+chrony:
```

Comment:
Let's keep the state name the same (`ntp`) but conditionally swap the package `- name: ` and service `- name:` depending on the OS. This will allow other states to be generic and not have to conditionally know whether to use `ntp` or `crony`